readthedocs/projects/tests/test_build_tasks.py

```diff
@@ -76,179 +69,199 @@ def _config_file(self, config):
         config = BuildConfigV2(
             {},
             config,
-            source_file='readthedocs.yaml',
+            source_file="readthedocs.yaml",
         )
         config.validate()
         return config
 
 
 class TestBuildTask(BuildEnvironmentBase):
-
     @pytest.mark.parametrize(
-        'formats,builders',
+        "formats,builders",
         (
-            (['pdf'], ['latex']),
-            (['htmlzip'], ['readthedocssinglehtmllocalmedia']),
-            (['epub'], ['epub']),
-            (['pdf', 'htmlzip', 'epub'], ['latex', 'readthedocssinglehtmllocalmedia', 'epub']),
-            ('all', ['latex', 'readthedocssinglehtmllocalmedia', 'nepub']),
-        )
+            (["pdf"], ["latex"]),
+            (["htmlzip"], ["readthedocssinglehtmllocalmedia"]),
+            (["epub"], ["epub"]),
+            (
+                ["pdf", "htmlzip", "epub"],
+                ["latex", "readthedocssinglehtmllocalmedia", "epub"],
+            ),
+            ("all", ["latex", "readthedocssinglehtmllocalmedia", "nepub"]),
+        ),
     )
-    @mock.patch('readthedocs.projects.tasks.builds.load_yaml_config')
+    @mock.patch("readthedocs.doc_builder.builder.load_yaml_config")
     @pytest.mark.skip
     def test_build_sphinx_formats(self, load_yaml_config, formats, builders):
-        load_yaml_config.return_value = self._config_file({
-            'version': 2,
-            'formats': formats,
-            'sphinx': {
-                'configuration': 'docs/conf.py',
-            },
-        })
+        load_yaml_config.return_value = self._config_file(
+            {
+                "version": 2,
+                "formats": formats,
+                "sphinx": {
+                    "configuration": "docs/conf.py",
+                },
+            }
+        )
 
         self._trigger_update_docs_task()
 
-        self.mocker.mocks['environment.run'].assert_any_call(
+        self.mocker.mocks["environment.run"].assert_any_call(
```

Comment:
It's unfortunate that this has a ton of linting changes. Did much change in the code itself?

Reply:
Yeah, we will be dealing with these kinds of changes because we are linting the files/sections we modify on each PRs.

To answer your questions, this file was not too much modified. However, some of the `mock.patch` paths were upgraded to match the new ones and some tests were adapted to check the new methods/functions. Nothing that needs a deep review, tho.